spec/support/stub_config_keys.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module StubConfigKeys
+  extend RSpec::SharedContext
+
+  before do
+    allow(Uploadcare.config).to receive_message_chain(:public_key).and_return('demopublickey')
+    allow(Uploadcare.config).to receive_message_chain(:secret_key).and_return('demosecretkey')
+  end
```

Comment:
This has been added due to changes from https://github.com/uploadcare/uploadcare-ruby/pull/161